packages/product-components/src/components/SelectAssetModal/SelectAssetModal.tsx

```diff
@@ -1,137 +1,184 @@
-import { useEffect, useState } from 'react';
+import { useMemo, useState } from 'react';
 import {
-    AssetLogo,
-    Button,
     Column,
     Icon,
     Input,
     NewModal,
-    Row,
-    useElevation,
     useScrollShadow,
     VirtualizedList,
-    Text,
 } from '@trezor/components';
 import { mapElevationToBackgroundToken, spacings } from '@trezor/theme';
 import { AssetItem } from './AssetItem';
+import { NetworkTabs } from './NetworkTabs';
+import { useIntl } from 'react-intl';
+import { AssetItemNotFound } from './AssetItemNotFound';
 
-const LIST_HEIGHT = '50vh';
-const height = 60;
+export interface SelectAssetOptionCurrencyProps {
+    type: 'currency';
+    value: string; // CryptoId (networkId + contractAddress)
+    label: string; // token shortcut
+    cryptoName: string | undefined; // full name
+    coingeckoId: string; // CryptoId (networkId)
+    contractAddress?: string; // CryptoId (contractAddress)
```

Comment:
Couldn't we do it without comments please? 🙏  ideally naming should be self-explanatory (and ideally we should keep consistency with AssetLogo).

Reply:
It is used to Select options, and I’d prefer to use value and label as the standard. On the other hand, I understand your point.